data_services/deploy/ansible/roles/forecastio/files/data_queues/forecastio/forecastio_air.py

```diff
@@ -0,0 +1,77 @@
+from __future__ import absolute_import
```

Comment:
Maybe we need two new repos then?:  

1) An ansible-focused repo with a role to deploy a geonode template project (ie geonodegp here).  This would include most of the code under the data_services folder

2) A repo for the data_queues code that would be packaged as a pluggable django app, and installed as part of 1) above.  This would mean modifying the following files in 1):
- Add the app as a requirement in https://github.com/Kitware/minerva/blob/minerva-geonode/data_services/deploy/ansible/roles/geonodegp/files/geonodegp/requirements.txt
- Add 'data_queues' to the INSTALLED_APPS setting in https://github.com/Kitware/minerva/blob/minerva-geonode/data_services/deploy/ansible/roles/geonodegp/files/geonodegp/geonodegp/settings.py

Ideally I'd like to keep the data_queues code separate from the ansible code.